packages/core/src/readers/index.ts

```diff
@@ -3,6 +3,7 @@ export * from "./CSVReader";
 export * from "./DocxReader";
 export * from "./HTMLReader";
 export * from "./ImageReader";
+export * from "./LlamaParseReader";
```

Comment:
@himself65 Contrary to the other readers, I didn't add an explicit export in package.json. Makes sense to me because a) it doesn't have any special dependencies and b) we probably always want to include this code